.github/workflows/ci.yml

```diff
@@ -39,144 +38,144 @@ env:
+#        uses: actions/cache@v1
+#        env:
+#          cache-name: cache-pre-commit
+#        with:
+#          path: ~/.cache/pre-commit
+#          key: ${{ env.cache-name }}-${{ github.job }}-${{ hashFiles('.pre-commit-config.yaml') }}
+#      - name: "Free space"
+#        run: ./scripts/ci/ci_free_space_on_ci.sh
+#      - name: "Build CI image"
+#        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+#      - name: "Static checks"
+#        run: |
+#          python -m pip install pre-commit \
+#              --constraint requirements/requirements-python${PYTHON_MAJOR_MINOR_VERSION}.txt
+#          ./scripts/ci/ci_run_static_checks.sh
+#
+#  docs:
+#    timeout-minutes: 60
+#    name: Build docs
+#    runs-on: ubuntu-latest
+#    env:
+#      CI_JOB_TYPE: "Documentation"
+#    steps:
+#      - uses: actions/checkout@master
+#      - name: "Build CI image ${{ matrix.python-version }}"
+#        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+#      - name: "Build docs"
+#        run: ./scripts/ci/ci_docs.sh
+#
+#  build-prod-image:
+#    timeout-minutes: 60
+#    name: "Build prod image Py${{ matrix.python-version }}"
+#    runs-on: ubuntu-latest
+#    strategy:
+#      matrix:
+#        python-version: [3.6, 3.7]
+#    env:
+#      PYTHON_MAJOR_MINOR_VERSION: ${{ matrix.python-version }}
+#      CI_JOB_TYPE: "Prod image"
+#    steps:
+#      - uses: actions/checkout@master
+#      - name: "Build PROD image ${{ matrix.python-version }}"
+#        run: ./scripts/ci/ci_build_production_images.sh
+#
+#  prepare-backport-packages:
+#    timeout-minutes: 60
+#    name: "Backport packages"
+#    runs-on: ubuntu-latest
+#    env:
+#      INSTALL_AIRFLOW_VERSION: "1.10.10"
+#      CI_JOB_TYPE: "Prepare & test packages"
+#      PYTHON_MAJOR_MINOR_VERSION: 3.6
+#    steps:
+#      - uses: actions/checkout@master
+#        with:
+#          fetch-depth: 0
+#      - uses: actions/setup-python@v1
+#        with:
+#          python-version: '3.x'
+#      - name: "Free space"
+#        run: ./scripts/ci/ci_free_space_on_ci.sh
+#      - name: "Build CI image ${{ matrix.python-version }}"
+#        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+#      - name: "Prepare & test backport packages"
+#        run: |
+#          ./scripts/ci/ci_prepare_and_test_backport_packages.sh
+#
+#  pyfiles:
+#    timeout-minutes: 10
+#    name: "Count changed important files"
+#    runs-on: ubuntu-latest
+#    outputs:
+#      count: ${{ steps.pyfiles.outputs.count }}
+#    steps:
+#      - uses: actions/checkout@master
+#      - name: "Get count of changed python files"
+#        run: |
+#          set +e
+#          ./scripts/ci/ci_count_changed_files.sh ${GITHUB_SHA} '\.py$|.github/workflows/|^Dockerfile'
+#          echo "::set-output name=count::$?"
+#        id: pyfiles
   tests-kubernetes:
     timeout-minutes: 80
-    name: "kubernetes-test-${{matrix.kube-mode}}-kube"
+    name: "K: ${{matrix.kube-mode}} ${{matrix.python-version}} ${{matrix.kubernetes-version}}"
```

Comment:
```suggestion
    name: "k8s: ${{matrix.kube-mode}} ${{matrix.python-version}} ${{matrix.kubernetes-version}}"
```

Reply:
Yeah. Those names are kind of tricky :). Too long and they get unreadable in GA interface in the panel on the left... Will improve it :).